internal/pkg/endpoints/kubernetes.go

```diff
@@ -270,7 +284,7 @@ func endpointsTarget(e *corev1.Endpoints, u url.URL) Target {
 func endpointsTargets(e *corev1.Endpoints, s *corev1.Service) []Target {
 	// we need to pass the service since the annotations are not inherited
 	port := getPort(s)
-	path := getPath(s)
+	path, query := parsePath(getPath(s))
```

Comment:
getPath name is still valid? should we update somehow the signature?

Reply:
Might me bending corners a little bit but as the label/annotation called `prometheus.io/path` I'm not super unhappy about the name.

I'd love to get rid of the `get` prefix but I did not want to refactor too much either :(

Reply:
fine for me